disabled compile from included plugins, we dont support this way on gentoo
support for linguas handling
fixed install an multicore cpu

changed path to install header files in own dir vdr, libsi
this have to fix, need some adaption in vdr-plugin-2.eclass and depended plugins, later...

Signed-of-by: Joerg Bornkessel <hd_brummy@gentoo.org> (2012/12/30)

diff -Naur vdr-1.7.35.orig/Makefile vdr-1.7.35/Makefile
--- vdr-1.7.35.orig/Makefile	2012-12-30 14:47:57.000000000 +0100
+++ vdr-1.7.35/Makefile	2012-12-30 14:48:30.000000000 +0100
@@ -101,7 +101,7 @@
 VDRVERSION = $(shell sed -ne '/define VDRVERSION/s/^.*"\(.*\)".*$$/\1/p' config.h)
 APIVERSION = $(shell sed -ne '/define APIVERSION/s/^.*"\(.*\)".*$$/\1/p' config.h)
 
-all: vdr i18n plugins
+all: vdr i18n
 
 # Implicit rules:
 
@@ -154,7 +154,7 @@
 
 PODIR     = po
 LOCALEDIR = locale
-I18Npo    = $(wildcard $(PODIR)/*.po)
+I18Npo    = $(foreach dir,$(LINGUAS),$(wildcard $(PODIR)/$(dir)*.po))
 I18Nmsgs  = $(addprefix $(LOCALEDIR)/, $(addsuffix /LC_MESSAGES/vdr.mo, $(notdir $(foreach file, $(I18Npo), $(basename $(file))))))
 I18Npot   = $(PODIR)/vdr.pot
 
@@ -177,7 +177,7 @@
 
 install-i18n:
 	@mkdir -p $(DESTDIR)$(LOCDIR)
-	@(cd $(LOCALEDIR); cp -r --parents * $(DESTDIR)$(LOCDIR))
+	@(cd $(LOCALEDIR); for linguas in $(LINGUAS); do [ "$$linguas" = "en" ] && continue; cp -r --parents $$linguas* $(DESTDIR)$(LOCDIR); done)
 
 # The 'include' directory (for plugins):
 
@@ -242,11 +242,12 @@
 
 install-dirs:
 	@mkdir -p $(DESTDIR)$(VIDEODIR)
-	@mkdir -p $(DESTDIR)$(CONFDIRDEF)
+#	@mkdir -p $(DESTDIR)$(CONFDIRDEF)
 	@mkdir -p $(DESTDIR)$(CACHEDIRDEF)
 	@mkdir -p $(DESTDIR)$(RESDIRDEF)
 
 install-conf:
+	@mkdir -p $(DESTDIR)$(CONFDIRDEF)
 	@cp -n *.conf $(DESTDIR)$(CONFDIRDEF)
 
 
@@ -268,8 +269,9 @@
 # Includes:
 
 install-includes: include-dir
-	@mkdir -p $(DESTDIR)$(INCDIR)
-	@cp -pLR include/vdr include/libsi $(DESTDIR)$(INCDIR)
+	@mkdir -p $(DESTDIR)$(INCDIR)/vdr $(DESTDIR)$(INCDIR)/vdr/libsi
+	@cp -pLR include/vdr $(DESTDIR)$(INCDIR)
+	@cp -pLR include/libsi Make.config $(DESTDIR)$(INCDIR)/vdr
 
 # pkg-config file:
 